src/databricks/labs/remorph/queries/tables/reconcile/main.sql

```diff
@@ -0,0 +1,18 @@
+CREATE OR REPLACE TABLE main (
+    recon_table_id BIGINT NOT NULL,
+    recon_id STRING NOT NULL,
+    source_type STRING NOT NULL,
+    source_table STRUCT<
+                         catalog: STRING NOT NULL,
+                         schema: STRING NOT NULL,
+                         table_name: STRING NOT NULL
+                        > NOT NULL,
+    target_table STRUCT<
+                         catalog: STRING NOT NULL,
```

Comment:
remove not null for the target_table catalog also .. we may face in this future anyway